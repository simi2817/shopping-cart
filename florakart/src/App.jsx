import Navigation from './components/Navigation';
import Header from './components/Header';

const App = () => {
  return (
    <div className="app">
      <Header />
      <Navigation />
      <br />
      <p>Welcome to <b>Flora-Kart</b>!</p>
      <br />
      <img src='https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExZDA1cnNwMWp3MHdrcGlvbDZhNGg4YTE0Nzh2NWZ5dDcxdjdwNncydyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/ckAhX7LBZrhst2mG2j/giphy.gif' alt='plant-gif' />
      <p>
        <b>Flora-Kart</b> helps you discover the best plants for your space, delivers them to your door and helps you look after them.
      </p>
    </div>
  )
}

export default App;
