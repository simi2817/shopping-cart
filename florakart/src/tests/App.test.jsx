import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';

import App from '../App.jsx';
import { MemoryRouter } from 'react-router-dom';

describe('App', () => {
  it('renders headline', () => {
    const { container } = render(<MemoryRouter><App /></MemoryRouter>);
    expect(container).toMatchSnapshot();
  });
});