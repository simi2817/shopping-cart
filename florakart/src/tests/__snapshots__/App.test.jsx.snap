// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > renders headline 1`] = `
<div>
  <div
    class="app"
  >
    <h1
      class="header"
    >
      Flora-Kart
    </h1>
    <div
      class="navigation"
    >
      <nav>
        <a
          class="navLink"
          href="/about"
        >
          About-Us
        </a>
        <a
          class="navLink"
          href="/categories"
        >
          Categories
        </a>
        <a
          class="navLink"
          href="/cart"
        >
          Cart
        </a>
      </nav>
    </div>
    <br />
    <p>
      Welcome to 
      <b>
        Flora-Kart
      </b>
      !
    </p>
    <br />
    <img
      alt="plant-gif"
      src="https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExZDA1cnNwMWp3MHdrcGlvbDZhNGg4YTE0Nzh2NWZ5dDcxdjdwNncydyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/ckAhX7LBZrhst2mG2j/giphy.gif"
    />
    <p>
      <b>
        Flora-Kart
      </b>
       helps you discover the best plants for your space, delivers them to your door and helps you look after them.
    </p>
  </div>
</div>
`;
